nilearn/masking.py

```diff
@@ -701,10 +701,7 @@ def compute_brain_mask(
         )
 
     resampled_template = cache(resampling.resample_to_img, memory)(
-        template,
-        target_img,
-        copy_header=True,
-        force_resample=False,  # TODO (nilearn >= 0.13.0) update to True
```

Comment:
Ok so the test failures are because of changing force_resample = False to True.

@Remi-Gau, do we have a good reason why we want to change to True as the default? 
All I could find so far was that the implementation with False is wrong ([here](https://github.com/nilearn/nilearn/pull/4412)), but not the reasoning behind it.